testbed_api/camera.py

```diff
@@ -7,30 +10,58 @@ class VideoCamera:
 
     def __init__(self, index: int = 0):
         self._capture = cv2.VideoCapture(index)
+        self._capture.set(cv2.CAP_PROP_FOURCC, cv2.VideoWriter_fourcc('X', 'V', 'I', 'D'))
```

Comment:
Choosing a codec can improve data transmission. The most suitable of the popular ones: XVID, X264. Refs: [stackoverflow](https://stackoverflow.com/questions/32353246/how-to-improve-opencv-and-python-videowriter-resolution)